From 74c6c159c2b499bdf3c39961bd40eb9243624226 Mon Sep 17 00:00:00 2001
From: Naushir Patuck <naush@raspberrypi.com>
Date: Wed, 29 Nov 2023 13:09:05 +0000
Subject: [PATCH] drivers: media: pisp_be: pisp_fe: Update UAPI header licenses

Update the license tags on the pisp UAPI header files with the
"Linux-syscall-note" clause.  Also replace the "GPL-2.0" tag with the
preferred "GPL-2.0-only" tag.

Signed-off-by: Naushir Patuck <naush@raspberrypi.com>
---
 drivers/media/platform/raspberrypi/pisp_be/pisp_be_config.h  | 2 +-
 drivers/media/platform/raspberrypi/rp1_cfe/pisp_common.h     | 2 +-
 drivers/media/platform/raspberrypi/rp1_cfe/pisp_fe_config.h  | 2 +-
 drivers/media/platform/raspberrypi/rp1_cfe/pisp_statistics.h | 2 +-
 drivers/media/platform/raspberrypi/rp1_cfe/pisp_types.h      | 2 +-
 5 files changed, 5 insertions(+), 5 deletions(-)

--- a/drivers/media/platform/raspberrypi/pisp_be/pisp_be_config.h
+++ b/drivers/media/platform/raspberrypi/pisp_be/pisp_be_config.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0-only */
+/* SPDX-License-Identifier: GPL-2.0-only WITH Linux-syscall-note */
 /*
  * PiSP Back End configuration definitions.
  *
--- a/drivers/media/platform/raspberrypi/rp1_cfe/pisp_common.h
+++ b/drivers/media/platform/raspberrypi/rp1_cfe/pisp_common.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0-only WITH Linux-syscall-note */
 /*
  * RP1 PiSP common definitions.
  *
--- a/drivers/media/platform/raspberrypi/rp1_cfe/pisp_fe_config.h
+++ b/drivers/media/platform/raspberrypi/rp1_cfe/pisp_fe_config.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0-only WITH Linux-syscall-note */
 /*
  * RP1 PiSP Front End Driver Configuration structures
  *
--- a/drivers/media/platform/raspberrypi/rp1_cfe/pisp_statistics.h
+++ b/drivers/media/platform/raspberrypi/rp1_cfe/pisp_statistics.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0-only WITH Linux-syscall-note */
 /*
  * RP1 PiSP Front End statistics definitions
  *
--- a/drivers/media/platform/raspberrypi/rp1_cfe/pisp_types.h
+++ b/drivers/media/platform/raspberrypi/rp1_cfe/pisp_types.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0-only WITH Linux-syscall-note */
 /*
  * RP1 PiSP Front End image definitions.
  *
